Web/DpAssignment1/designPattern1qn1/Logger.java

```diff
@@ -0,0 +1,48 @@
+package designPattern1qn1;
+
+import java.io.IOException;
+import java.util.logging.FileHandler;
+import java.util.logging.SimpleFormatter;
+
+public class Logger {
+
+	
+	private java.util.logging.Logger jLogger= java.util.logging.Logger.getLogger("MyLog");
```

Comment:
I think you should have created your own logger rather than using what Java provides as per the assignment. Was it told like this only?